src/Chapter2.hs

```diff
@@ -740,8 +753,12 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
-
+contains :: Int -> [[Int]] -> [[Int]]
+contains n l = go [] l
+  where
+    go :: [[Int]] -> [[Int]] -> [[Int]]
+    go acc [] = reverse acc
+    go acc (x:xs) = if n `elem` x then go (x:acc) xs else go acc xs
```

Comment:
Nice implementation with the Recursive go pattern!

In this particular function, you can use standard function `filter` to keep only those elements, satisfying the given check